File: benchmarks/python/loop_sum.py
import os
import sys

def sum_to_iter(n: int) -> int:
    s = 0
    i = 0
    while i < n:
        s += i
        i += 1
    return s

def main():
    N = int(os.getenv("N", "500000000"))
    x = sum_to_iter(N)
    print(x)
    _ = x

if __name__ == "__main__":
    main()

File: benchmarks/c/loop_sum.c
#include <stdint.h>
#include <stdio.h>
#include <stdlib.h>

static int64_t sum_to_iter(int64_t n){
    int64_t s = 0;
    for (int64_t i = 0; i < n; i++) {
        s += i;
    }
    return s;
}

int main(){
    const char* env = getenv("N");
    int64_t N = env ? atoll(env) : 500000000;
    int64_t x = sum_to_iter(N);
    printf("%lld\n", (long long)x);
    return 0;
}
